tests/docker-entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -xe
+cd tests
+cat <<EOF |
```

Comment:
Instead of `cat` I think you should use `python -c` otherwise it will be just printed to stdout